.travis.yml

```diff
@@ -21,4 +21,4 @@ script:
   - export JBOSS_FOLDER=$PWD
   - export JBOSS_VERSION=13.0.0.Final
   - cd ../..
-  -travis_wait mvn -q clean install -Dwildfly-release-13.0.0.Final -Dwildfly-release-13.0.0.Final-jdk8 -Dstandalone
+  - mvn -q clean install -Dwildfly-release-13.0.0.Final -Dwildfly-release-13.0.0.Final-jdk8 -Dstandalone
```

Comment:
Very good. travis_wait is not needed any more. This is used only when the program does not produces an output for a time interval. Here, in the past it was used to bypass the logging size limit, which is not needed any more as we use the -q parameter.